devel/fortio/README.md

```diff
@@ -0,0 +1,127 @@
+# Φορτίο
+
+Φορτίο (fortio) is [Istio](https://istio.io/)'s load testing tool. Fortio runs at a specified query per second (qps) and records an histogram of execution time and calculates percentiles (e.g. p99 ie the response time such as 99% of the requests take less than that number (in seconds, SI unit))
+
+The name fortio comes from greek φορτίο which is load/burden.
+
+## Command line arguments
+
+```
+$ fortio
+Φορτίο 0.1 usage:
+
+fortio [flags] url
+  -H value
+    	Additional Header(s)
+  -c int
+    	Number of connections/goroutine/threads (0 doesn't change internal default)
```

Comment:
this is really the number of goroutines to start for making connections right?  should we just be explicit about that?

as far as I can tell, it doesn't enforce an explicit number of open connections and it isn't threads related.